hw/BUILD

```diff
@@ -35,7 +35,10 @@ fusesoc_build(
         "binary": ["sim-verilator/Vchip_sim_tb"],
     },
     systems = ["lowrisc:dv:chip_verilator_sim"],
-    tags = ["verilator"],
+    tags = [
+        "manual",
+        "verilator",
```

Comment:
FYI, the verilator tag wasn't filtering it out of the build because --test_tag_filters doesn't affect what's built, and it's building all the targets that match the wildcard because of --build_tests_only=false